specification/storage/resource-manager/Microsoft.Storage/stable/2021-06-01/storage.json

```diff
@@ -0,0 +1,4771 @@
+        "bypass": {
+          "type": "string",
+          "enum": [
+            "None",
+            "Logging",
+            "Metrics",
+            "AzureServices"
+          ],
+          "x-ms-enum": {
+            "name": "Bypass",
+            "modelAsString": true
+          },
+          "x-ms-client-name": "Bypass",
+          "default": "AzureServices",
+          "description": "Specifies whether traffic is bypassed for Logging/Metrics/AzureServices. Possible values are any combination of Logging|Metrics|AzureServices (For example, \"Logging, Metrics\"), or None to bypass none of those traffics."
+        },
+        "resourceAccessRules": {
+          "type": "array",
+          "items": {
+            "description": "Resource Access Rules.",
+            "$ref": "#/definitions/ResourceAccessRule"
+          },
+          "description": "Sets the resource access rules"
+        },
+        "virtualNetworkRules": {
+          "type": "array",
+          "items": {
+            "description": "Virtual Network rule.",
+            "$ref": "#/definitions/VirtualNetworkRule"
+          },
+          "description": "Sets the virtual network rules"
+        },
+        "ipRules": {
+          "type": "array",
+          "items": {
+            "description": "IP rule with specific IP or IP range in CIDR format.",
+            "$ref": "#/definitions/IPRule"
+          },
+          "description": "Sets the IP ACL rules"
+        },
+        "defaultAction": {
+          "type": "string",
+          "enum": [
+            "Allow",
+            "Deny"
+          ],
+          "x-ms-enum": {
+            "name": "DefaultAction",
+            "modelAsString": false
+          },
+          "default": "Allow",
+          "description": "Specifies the default action of allow or deny when no other rules match."
+        }
+      },
+      "required": [
+        "defaultAction"
+      ],
+      "description": "Network rule set"
+    },
+    "AzureFilesIdentityBasedAuthentication": {
+      "properties": {
+        "directoryServiceOptions": {
+          "type": "string",
+          "description": "Indicates the directory service used.",
+          "enum": [
+            "None",
+            "AADDS",
+            "AD"
+          ],
+          "x-ms-enum": {
+            "name": "DirectoryServiceOptions",
+            "modelAsString": true
+          }
+        },
+        "activeDirectoryProperties": {
+          "$ref": "#/definitions/ActiveDirectoryProperties",
+          "description": "Required if choose AD."
+        },
+        "defaultSharePermission": {
+          "type": "string",
+          "description": "Default share permission for users using Kerberos authentication if RBAC role is not assigned.",
+          "enum": [
+            "None",
+            "StorageFileDataSmbShareReader",
+            "StorageFileDataSmbShareContributor",
+            "StorageFileDataSmbShareElevatedContributor",
+            "StorageFileDataSmbShareOwner"
```

Comment:
Per feature team, "StorageFileDataSmbShareOwner" an unsupported value in SRP , We have laready remove it from PSH/CLI. Should we also remove it from swagger?